src/common/services/api/order.service.js

```diff
@@ -352,6 +352,27 @@ class Order {
     return cartData
   }
 
+  storeBrandedCoverFeeDecision (coverFees) {
+    this.localStorage.setItem('brandedCoverFees', angular.toJson(coverFees))
```

Comment:
I saw in your comment that you purposely had this using a different key. Seems like branded and give.cru.org should always be running on a different domain and local storage won't be shared (except maybe in development where it shouldn't matter.

If we think there's a reason to keep it seperate, that's fine, just seems like more complexity and code to maintain that we could potentially reuse.

Reply:
That makes sense. My only hesitation is the testing that FDS will be doing in stage. Hitting S3 directly is not really usable for user acceptance testing, and I've put a small branded checkout directly onto give-stage2. This would have conflicting values when going between branded and standard checkout. But, as you say, it is just for testing.

Reply:
Going to change it to use the regular cover-fees local storage.